frontend/src/employee-frontend/components/child-information/assistance-need/decision/common.tsx

```diff
@@ -2,13 +2,65 @@
 //
 // SPDX-License-Identifier: LGPL-2.1-or-later
 
+import React from 'react'
 import styled from 'styled-components'
 
+import { useTranslation } from 'employee-frontend/state/i18n'
+import { AssistanceNeedDecisionStatus } from 'lib-common/generated/api-types/assistanceneed'
+import { StaticChip } from 'lib-components/atoms/Chip'
+import { FixedSpaceColumn } from 'lib-components/layout/flex-helpers'
+import { Strong } from 'lib-components/typography'
 import { defaultMargins } from 'lib-components/white-space'
+import { theme } from 'lib-customizations/common'
 
 export const FooterContainer = styled.div`
   display: flex;
   justify-content: flex-end;
   align-items: center;
   padding: ${defaultMargins.s};
 `
+
+export const AssistanceNeedDecisionStatusChip: React.FC<{
```

Comment:
Looking at the codebase of the frontend, `React.FC` is not really used; instead, `const X = React.memo(function X(…) { … })` is. Maybe this component and the one below could use the memo approach as well?